src/operations.json

```diff
@@ -0,0 +1,20 @@
+{
+	"insert-ai-generated-content": {
+		"label": "Insert",
+		"tooltip": "Insert the generated content into the document",
+		"steps": [
+			{
+				"type": "transform/setContextNodeIdToSelectedElement"
```

Comment:
Where are these two transform steps for?
The "insert-text" doesn't require a contextNodeId or documentId I believe